airbyte-cdk/python/airbyte_cdk/sources/concurrent_source/partition_generation_completed_sentinel.py

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+from airbyte_cdk.sources.streams.concurrent.abstract_stream import AbstractStream
+
+
+class PartitionGenerationCompletedSentinel:
```

Comment:
this used to be a singleton object but I created a new class with a pointer on the stream since ConcurrentSource reads multiple streams concurrently